package/MDAnalysis/analysis/bat.py

```diff
@@ -372,7 +372,7 @@ def _single_frame(self):
         # The rotation axis is a normalized vector pointing from atom 0 to 1
         # It is described in two degrees of freedom
         # by the polar angle and azimuth
-        if (self._root.dimensions[:3] == 0).any():
+        if self._root.dimensions is None:
```

Comment:
I think `(self._root.dimensions[:3] == 0).any()` asks if any of x, y, z are 0, not just if it's unassigned. Is this a pertinent difference @daveminh ?

Reply:
`== 0) .any()` was the old idiom for unassigned.  This keeps the behaviour the same.

Reply:
I'm not seeing any other match to that pattern in `develop`? I'd agree with @lilyminium that this is either intentional or it was a typo to begin with.

pinging @daveminh again.

Reply:
raised #3326